notebooks/107-speech-recognition-quantization/107-speech-recognition-quantization-data2vec.ipynb

```diff
@@ -0,0 +1,682 @@
+{
```

Comment:
<p>typos: "min", "qunatization", "preformance", </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/785/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>min is not typo, but anyway, I rephrase to make message more clear</p>

Reply:
<p>shouldn't be keep in mind?</p>

Reply:
no, I mean to not bring a lot of changes, "to keep it min"